sdk/personalizer/azure-ai-personalizer/tests/test_configuration.py

```diff
@@ -0,0 +1,72 @@
+from devtools_testutils import AzureRecordedTestCase
+import helpers
+import time
+
+
+def configuration_equals(actual, expected):
+    assert actual.get('modelExportFrequency') == expected.get('modelExportFrequency')
+    assert actual.get('defaultReward') == expected.get('defaultReward')
+    assert actual.get('modelRetrainDays') == expected.get('modelRetrainDays')
+    assert actual.get('rewardWaitTime') == expected.get('rewardWaitTime')
+    assert actual.get('explorationPercentage') == expected.get('explorationPercentage')
+    assert actual.get('logRetentionDays') == expected.get('logRetentionDays')
+
+
+def policy_equals(actual, expected):
+    assert actual.get('arguments') == expected.get('arguments')
+
+
+class TestConfiguration(AzureRecordedTestCase):
+
+    @helpers.PersonalizerPreparer()
+    def test_update_configuration(self, **kwargs):
+        personalizer_endpoint = kwargs.pop('personalizer_endpoint_single_slot')
+        personalizer_api_key = kwargs.pop('personalizer_api_key_single_slot')
+        client = helpers.create_personalizer_client(personalizer_endpoint, personalizer_api_key)
+        configuration = {
+            "rewardAggregation": "average",
+            "modelExportFrequency": "PT3M",
+            "defaultReward": 1.0,
+            "modelRetrainDays": 0,
+            "rewardWaitTime": "PT4H",
+            "explorationPercentage": 0.3,
+            "logRetentionDays": -1,
+            "learningMode": "Online",
+        }
+        updated_configuration = client.service_configuration.update(configuration)
+        configuration_equals(configuration, updated_configuration)
+        self.sleep(30)
+        new_configuration = client.service_configuration.get()
```

Comment:
Is `update` a long-running operation? Do we expect users to need to sleep between the update/get calls?

Reply:
Update is not an LRO. The reason we wait for some time is that there is a lag in updating the cache objects on the different nodes that host the service and this lag will allow for those objects to be updated. We have some ongoing work to accommodate sticky sessions when this wait will not be required after that is implementated.